pkg/controller/schedulingmanager/controller.go

```diff
@@ -20,21 +20,16 @@ import (
 	"github.com/golang/glog"
 	"github.com/pkg/errors"
 
+	"k8s.io/apimachinery/pkg/util/runtime"
```

Comment:
(No action required) I'm not going to insist in this instance since the PR is large and I'm not sure how easy it is for you to split it apart (I have tools that make it easy), but in future please try to separate non-functional changes like import ordering into separate commits for a cleaner history.

Reply:
got your point. would keep in mind in future pr's.